README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are required to comply with the following code of conduct. HangOps admins are responsible for ensuring that all community members follow this code within HangOps spaces.
+
+## The Short Version
+
+Be respectful of other people; respectfully ask people to stop if you are bothered; and if you can’t resolve an issue contact the admins. If you are unable to follow our Community Code of Conduct problem, it will be apparent and you may be asked to leave HangOps.
+
+## The Long Version
+
+### Respect
+
+HangOps is a community of people who care about running production systems reliably. We have diverse backgrounds, skills, talents, experiences, cultures, and opinions. We are an inclusive environment and we treat all individuals respectfully.
+
+We value respectful behavior above individual opinions.
+
+Respectful behavior includes:
+
+* Be considerate, kind, constructive, and helpful.
+* Avoid demeaning, discriminatory, harassing, hateful, or physically threatening behavior, speech, and imagery.
+* If you’re not sure, ask someone instead of assuming. No, really. Just ask the admins.
+* Model respectful behaviour, but also address disrespect when you see it. One of the best ways to create a respectful community is to speak up when you see disrespectful behavior—and not just rely on admins to act!
+
+### Resolve Peacefully
+
+If you see someone behaving disrespectfully, we urge you to respectfully dissuade them from such behavior.
+
+If you experience disrespectful behavior toward yourself or anyone else and feel in any way unable or unwilling to respond or resolve it respectfully (for any reason), please immediately bring it to the attention of the admins who will listen and work to resolve the matter.
+
+### Apologize for Mistakes
+
+Should you catch yourself behaving disrespectfully, or if someone calls you out on disrespectful behavior, please listen intently, own up to your words and actions, and apologize accordingly. No one is perfect, and even well-intentioned people make mistakes. What matters is how you handle them and that you avoid repeating them.
+
+### Consequences
+
+If the admins determine that someone is behaving disrespectfully, the admins may take any action they deem appropriate within HangOps, up to and including expulsion and exclusion from HangOps.
+
+Admins will seek to resolve conflicts peacefully and in a manner that is positive for the community. The admins can’t foresee every situation, and thus if in the admin's judgment the best thing to do is to ask a disrespectful individual to leave, they will do so.
```

Comment:
> ask a disrespectful individual to leave

recommend stronger wording to reinforce the previous paragraph: 

"expel a disrespectful individual"